File: 1323-maximum-69-number/1323-maximum-69-number.py
class Solution:
    def maximum69Number (self, num: int) -> int:
        
        nums = str(num)
        ret = [] 
        mode =0 
        for i in range(len(nums)):
             
            if nums[i] =='6' and mode==0:
                ret.append( '9' ) 
                mode +=1 
            else:
                ret.append(nums[i])
        return int(''.join(ret))

File: 1323-maximum-69-number/1323-maximum-69-number.cpp
class Solution {
public:
    int maximum69Number (int num) {
        
        int x = num; 
        int six = -1 ; 
        int digit = 0; 

        while (x >0){
            if(x % 10 ==6){
                six = digit; 
            }
            x /=10 ; 
            digit++; 
        }

        return six == -1 ? num : num + 3*(int)pow(10, six) ; 
    }
};